src/components/activity/ActivityList.js

```diff
@@ -0,0 +1,51 @@
+import styled from 'styled-components';
+
+function ActivityList({activities, onHandleDelete}) {
+  const handleDelete = event => {
+    const activityId = event.target.dataset.id;
+    onHandleDelete(activityId);
+  };
+  return (
+    <>
+      <div>
+        <label>Aufgaben des Tages</label>
+      </div>
+      {activities.map(activity => (
+        <div key={activity.id}>
+          <StyledActivitiList>
```

Comment:
```suggestion
          <StyledActivityList>
```
But it's not a list, is it?